Packs/GoogleChronicleBackstory/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Chronicle",
     "description": "Retrieve Chronicle detections, impacted assets, IOC matches, and 3P alerts to enrich your XSOAR workflows.",
     "support": "partner",
-    "currentVersion": "4.0.1",
+    "currentVersion": "4.0.2",
```

I believe a minor version bump is more suitable.
```suggestion
    "currentVersion": "4.1.0",
```